### PR TITLE
content/*/integrations/memcached: add PyMemcache

### PR DESCRIPTION
**Pull request by odeke-em**
Add the OpenCensus-trace-instrumented fork of Pinterest's
PyMemcache at https://github.com/opencensus-integrations/pymemcache

as well as a demo app and screenshots.

### CODE REVIEW

**Comment by hvent90** [comment]
Awesome!

Table of contents needs a little adjustment to match the contents of the article. Here is a diff demonstrating the changes needed: https://github.com/hvent90/opencensus-website/commit/78a5962ffc38df020483a79656ef7ac3162bf32b

File: content/guides/integrations/memcached/Python.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+date: 2018-08-24T01:00:08-07:00
+draft: false
+aliases: [/integrations/memcached/python]
+---
+
+![memcached logo](/img/memcached-python.png)
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Using it](#using-it)
```
Comment (songy23) [inline content/guides/integrations/memcached/Python.md]: Please add the sub-tabs, as @hvent90 suggested.

File: content/guides/integrations/memcached/Python.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+- [Examining your traces](#enabling-your-traces)
+- [References](#references)
+
+## Introduction
+[Memcached](https://memcached.org) is one of the most used server caching and scaling technologies.
+
+It was created by [Brad Fitzpatrick](https://en.wikipedia.org/wiki/Brad_Fitzpatrick) in 2003 as a
+solution to scale his social media product [Live Journal](https://www.livejournal.com/)
+
+Pinterest's Python [PyMemcache](https://github.com/pinterest/pymemcache) client has been forked and [trace instrumented with OpenCensus](https://github.com/opencensus-integrations/pymemcache)
```
Comment (songy23) [inline content/guides/integrations/memcached/Python.md]: nit: trace instrumented with OpenCensus -> instrumented with OpenCensus tracing

File: content/guides/integrations/memcached/Python.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    main()
+{{</highlight>}}
+
+## Enabling OpenCensus
+To provide observability, we'll enable tracing by importing OpenCensus Python's tracing package
+
+{{<highlight python>}}
+from opencensus.trace.tracer import Tracer
+
+# For demo purposes, we'll always sample
```
Comment (songy23) [inline content/guides/integrations/memcached/Python.md]: nit: I think this is a h3 header `###`

File: content/guides/integrations/memcached/Python.md (hunk 2 above)
Comment (odeke-em) [reply content/guides/integrations/memcached/Python.md]: Roger that thanks!

File: content/guides/integrations/memcached/Python.md (hunk 1 above)
Comment (odeke-em) [reply content/guides/integrations/memcached/Python.md]: Roger that, thanks @hvent90 and @songy23!

File: content/guides/integrations/memcached/Python.md (hunk 3 above)
Comment (odeke-em) [reply content/guides/integrations/memcached/Python.md]: Oh, this is part of the source code :)
screen shot 2018-08-27 at 11 14 10 am

**Comment by odeke-em** [comment]
Thanks for the updates and reviews @hvent90 and @songy23., PTAL.

File: content/guides/integrations/memcached/Python.md (hunk 1 above)
Comment (odeke-em) [reply content/guides/integrations/memcached/Python.md]: I've updated that code, PTAL.

Comment (songy23) [reply content/guides/integrations/memcached/Python.md]: LGTM

**Pull request merged**